YOLO.xcodeproj/project.pbxproj

```diff
@@ -379,7 +359,7 @@
 				ASSETCATALOG_COMPILER_INCLUDE_ALL_APPICON_ASSETS = NO;
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 0;
-				DEVELOPMENT_TEAM = 3MR4P6CL3X;
+				DEVELOPMENT_TEAM = "";
```

Comment:
Same as above it must be:
```suggestion
				DEVELOPMENT_TEAM = 3MR4P6CL3X;
```

Reply:
I reverted DEVELOPMENT_TEAM to original.